docs/configuration/loots.md

```diff
@@ -0,0 +1,25 @@
+# Loots
```

Comment:
I'd consider renaming this to crates or boxes, since loot is generally used in the context of currencies